.dev_scripts/config_list.yaml

```diff
@@ -0,0 +1,23 @@
+hrnet_w32_coco_256x192:
```

Comment:
It's better to use a more specific filename like "benchmark_regression_cfg.yaml".

I suggest make the model list a part of the this yaml file, so it can be easily extended to contain new contents in the future.